ext/pcntl/tests/pcntl_exec_004.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+pcntl_exec() - Testing error when non-stringable arguments are passed for $args and $env_vars.
+--EXTENSIONS--
+pcntl
+--SKIPIF--
+<?php
+if (!getenv("TEST_PHP_EXECUTABLE") || !is_executable(getenv("TEST_PHP_EXECUTABLE"))) die("skip TEST_PHP_EXECUTABLE not set");
+?>
+--FILE--
+<?php
+try {
+    pcntl_exec(getenv("TEST_PHP_EXECUTABLE"), ['-n', new stdClass()]);
+} catch (Error $error) {
+    echo $error->getMessage() . "\n";
+}
+
+try {
+    pcntl_exec(getenv("TEST_PHP_EXECUTABLE"), ['-n'], [new stdClass()]);
+} catch (Error $error) {
+    echo $error->getMessage() . "\n";
+}
+
+echo "ok\n";
```

Comment:
While this doesn't hurt, it is no longer necessary either. It was good practice before run-tests.php has been improved to report crashes. I suggest to remove it (and also from the `--EXPECT--` section):
```suggestion
```

Reply:
I will change this. Push coming soon

Reply:
@cmb69 - change done :)